modules/rosa-configure.adoc

```diff
@@ -4,16 +4,16 @@
 // * rosa_cli/rosa-get-started-cli.adoc
 
 [id="rosa-configure_{context}"]
-= Configuring the rosa CLI
+= Configuring the {product-title} (ROSA) CLI (`rosa`)
 
-Use the following commands to configure the `rosa` CLI.
+Use the following commands to configure the {product-title} (ROSA) CLI (`rosa`).
 
 [id="rosa-login_{context}"]
 == login
 
 Log in to your Red Hat account, saving the credentials to the `rosa` configuration file. You must provide a token when logging in. You can copy your token from link:https://console.redhat.com/openshift/token/rosa[the {product-title} token page].
 
-The `rosa` CLI looks for a token in the following priority order:
+The {product-title} (ROSA) CLI (`rosa`) looks for a token in the following priority order:
```

Comment:
This and the next instance can just be ROSA CLI

Reply:
Maybe "ROSA CLI (rosa)" as this is the first instance and then later "ROSA CLI".